src/test/java/AlertServiceTest.java

```diff
@@ -0,0 +1,123 @@
+import dev.Roach.AlertService;
+import dev.Roach.datamodel.game.GamePojo;
+import dev.Roach.fetchers.GamesFetcher;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class AlertServiceTest {
+
+    @Mock
+    private HttpClient mockClient;
+    @Mock
+    private GamesFetcher mockGamesFetcher;
+
+    @InjectMocks
+    private AlertService alertService;
+
+    @BeforeEach
+    public void setUp() {
+        alertService = new AlertService();
+        alertService.setHttpClient(mockClient);
+    }
+
+    @Test
+    public void testCreateOrUpdateAlertShouldReturnCorrectInput() throws IOException, InterruptedException {
+
+        HttpResponse<String> mockResponse = Mockito.mock(HttpResponse.class);
+
+        when(mockResponse.body()).thenReturn("true");
+        when(mockClient.send(any(HttpRequest.class),
+                any(HttpResponse.BodyHandlers.ofString().getClass()))).thenReturn(mockResponse);
+
+        boolean result = alertService.createOrUpdateAlert("test@example.com", 1, 9.99);
```

Comment:
Please change name from result to something more descriptive, does this mean alert has been created?

Reply:
Changed variables to be more descriptive.